realtime_tools/CMakeLists.txt

```
@@ -80,9 +80,10 @@ if(BUILD_TESTING)
   target_link_libraries(realtime_buffer_tests realtime_tools)
 
   ament_add_gmock(lock_free_queue_tests test/lock_free_queue_tests.cpp)
+
   target_link_libraries(lock_free_queue_tests realtime_tools Boost::boost)
-  if(UNIX)
-    target_link_libraries(lock_free_queue_tests atomic)
+  if(UNIX AND NOT APPLE)
+    target_link_libraries(lock_free_queue_tests realtime_tools atomic)
```

```suggestion
    target_link_libraries(lock_free_queue_tests atomic)
```
realtime_tools is linked above